run_cls.py

```diff
@@ -58,7 +58,10 @@ def launch_cls(data, queue, njobs, nc, mem, wsp=False, fiducial=False, onlogin=F
     outdir = data.data['output']
     if fiducial:
         outdir = os.path.join(outdir, 'fiducial')
-    qjobs = get_queued_jobs()
+    try:
+        qjobs = get_queued_jobs()
```

Comment:
Better than let it fail when running outside of Glamdring would be to add an `if`-`else` statement. You can use, e.g.
```
if os.uname[1] == 'glamdring':
    qjobs = get_queued_jobs()
else:
    qjobs = ''
```

Reply:
Nice! Done